client/src/interface/Board.ts

```diff
@@ -1,18 +1,22 @@
-export interface Card{
-    _id: string;
-    name: string;
-    color: ColorTags;
-    deadline?: string;
+export interface Card {
+  _id?: string;
+  name: string;
+  colorCode: ColorTags;
+  deadline?: string;
+  attachments?: string[];
+  tags?: string[];
 }
+
 export interface Column {
-    _id: string;
-    cards: Card[];
-    name: string;
+  _id?: string;
```

Comment:
Sorry, May I know why you made _id optional?

Reply:
There shouldn't be an instance where id is undefined or null, we should ensure this is always there and enforce it